### ISSUE TITLE
EDA & insights

### ISSUE THREAD

**Issue by venkatnadella** [venkatnadella/Orders_in_Ontario#3]
This part needs careful understanding of the clean data and drawing some useful initial insights. These insights helps us in paving a way for our search towards the final goal. 
We expect the following tasks to be performed at this stage. 
- Custom Visualizations using Python
- Dedicated Dashboard

**Comment by venkatnadella** [comment]
@Amulya-9374 , @bajanb2021 , @tejasdhatrika Could you guys please look into this and send for code review to me.

**Comment by tejasdhatrika** [comment]
Yes, I have worked on the Exploratory Data Analysis and Insights specifically on correlation matrix, pair plots and column charts.
